workers/unity/Packages/io.improbable.gdk.debug/NetStatsViewer/NetStatViewer.cs

```diff
@@ -1,23 +1,28 @@
 using System.Collections.Generic;
 using System.Linq;
-using Improbable.Gdk.Core;
 using Improbable.Gdk.Core.NetworkStats;
 using Unity.Entities;
 using UnityEditor;
 using UnityEditor.UIElements;
 using UnityEngine.UIElements;
 
-namespace Improbable.Gdk.Debug
+namespace Improbable.Gdk.Debug.NetStats
 {
     internal class NetStatViewer : EditorWindow
     {
-        private const string WindowUxmlPath = "Packages/io.improbable.gdk.debug/NetStatsViewer/Templates/NetStatsWindow.uxml";
-        private const string UpdateRowUxmlPath = "Packages/io.improbable.gdk.debug/NetStatsViewer/Templates/UpdateRow.uxml";
+        private enum TabType
+        {
+            Updates,
+            Commands,
+            WorldCommands
+        }
+
+        private const string WindowUxmlPath =
+            "Packages/io.improbable.gdk.debug/NetStatsViewer/Templates/NetStatsWindow.uxml";
```

Comment:
use `Common.GetPackagePath`  to get the package path?

Reply:
I'm not sure if we should, it seems Unity uses this style internally as well and seems to automatically translate the path for downloaded packages.

Reply:
Using this method allows it to reload as well

Reply:
Yeah I think this isn't _actually_ a real path.